packages/manager/modules/pci/src/projects/project/serving/namespace/models/models.constants.js

```diff
@@ -0,0 +1,44 @@
+
+export const PROCESSING_API_STATUS = [
+  'pending',
+  'starting',
+  'scaling',
+];
+
+export const API_STATUS = {
+  pending: 'pending',
+  starting: 'starting',
+  running: 'running',
+  scaling: 'scaling',
+  waking: 'waking',
+  sleeping: 'sleeping',
+};
+
+export const VERSION_STATUS = {
+  pending: 'pending',
+  building: 'building',
+  built: 'built',
+  'build-error': 'build-error',
+  deploying: 'deploying',
+  deployed: 'deployed',
+  rollback: 'rollback',
+  failed: 'failed',
+};
+
+export const PROCESSING_VERSION_STATUS = [
+  'pending',
```

Comment:
Sometimes you are using an array, sometimes an object, looks like you are coding according to what you need right now instead of having an uniform way of doing things 🙂